tests/ops/op_math/test_dot.py

```diff
@@ -68,7 +65,7 @@ def test_dot_autograd(self):
         o = [qml.PauliX(0), qml.PauliY(1), qml.PauliZ(2)]
         op_sum = dot(c, o)
         op_sum_2 = Sum(
-            qml.PauliX(0),
+            SProd(qml.numpy.array(1.0), qml.PauliX(0)),
```

Comment:
Is this cosmetics or did the behavior change?

Reply:
It did change! Before we had an if-statement to avoid returning an `SProd` operator when `coeff = 1`. But this is not jittable, thus I removed it.

This small change shouldn't affect performance.

Reply:
Thanks!